app/manager-frontend/src/components/ConfigureResultsModifierModal.jsx

```diff
@@ -0,0 +1,160 @@
+    { value: { Id: "Type2", Limit: "Limit2" }, text: "Type2" },
+  ];
+
+  const handleSubmit = async (values, actions) => {
+    try {
+      // convert all empty strings to null
+      const payload = Object.entries(values).reduce(
+        (a, [k, v]) => ({
+          ...a,
+          [k]: v !== "" ? v : null,
+        }),
+        {}
+      );
+      // post to the api
+      await action({ values: payload }).json();
+      onClose();
+    } catch (e) {
+      console.error(e);
+      onConfirmClose();
+      setFeedback("Something went wrong!");
+      window.scrollTo(0, 0);
+    }
+    actions.setSubmitting(false);
+  };
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          {initialData ? "Update Results Modifier" : "Create Results Modifier"}
+        </ModalHeader>
+        <ModalCloseButton />
+        <ModalBody>
+          <Formik
+            onSubmit={handleSubmit}
+            initialValues={
+              initialData
+                ? {
+                    Order: initialData.order,
+                    Type: initialData.type,
+                    // capitalise the object keys in the parameters object
+                    Parameters: Object.fromEntries(
+                      Object.entries(initialData.parameters).map(([k, v]) => [
+                        k.charAt(0).toUpperCase() + k.slice(1),
+                        v,
+                      ])
+                    ),
+                  }
+                : {
+                    Order: "",
+                    Type: typeOptions[0].value,
+                    Parameters: {},
+                  }
+            }
+            validationSchema={validationSchema()}
+          >
+            {({ isSubmitting, values }) => (
+              <Form noValidate>
+                <VStack align="stretch" spacing={8}>
+                  {feedback && (
+                    <Alert status="error">
+                      <AlertIcon />
+                      {feedback}
+                    </Alert>
+                  )}
+                  <FormikInput label="Order" name={"Order"} type="number" />
+                  <FormikSelect
+                    label="Type"
+                    name={"Type"}
+                    type="Type"
+                    options={typeOptions}
+                  />
+                  <LowNumberSuppressionParameters type={values.Type} />
+                  <Button
+                    w="full"
+                    leftIcon={<FaArrowRight />}
+                    colorScheme="blue"
+                    onClick={onConfirmOpen}
+                    disabled={isSubmitting}
+                    isLoading={isSubmitting}
+                  >
+                    {initialData
+                      ? "Update Results Modifier"
+                      : "Create Results Modifier"}
+                  </Button>
+                  <Modal isOpen={isConfirmOpen} onClose={onConfirmClose}>
```

Comment:
It looks like this with the double modal 
<img width="960" alt="image" src="https://user-images.githubusercontent.com/38753056/178460282-297ca234-858d-4ab8-8848-caf373403bfc.png">

Reply:
hmm, not amazing, but i think it's ok.

However I'm not sure we need confirmation for creating; only editing.

If you mis-create something you can just delete it, but if you mis-edit something, you might not remember what you changed it from.

With that in mind, it might be nice for the confirmation modal to detail the changes.

Maybe a slim table:
| Parameter | Previous | New |
|-|-|-|
| Threshold | 5 | 10 |

make the new value bold and green or something ¯\_(ツ)_/¯

Reply:
Confirmation now only shows up for editing, and the diff table is shown for editing like this 
<img width="549" alt="image" src="https://user-images.githubusercontent.com/38753056/178493876-0cac3f98-f990-4e15-bf0b-8d38ba548f6b.png">